apartment.rb

```diff
@@ -1,2 +1,29 @@
+require_relative 'room'
+
 class Apartment
+
+  attr_reader :monthly_rent, :rooms
+
+  def initialize(args = {})
+    @number = args.fetch(:number)
+    @rooms = args.fetch(:rooms)
+    @monthly_rent = args.fetch(:monthly_rent, 1000)
+  end
+
+  def total_sqft
+    rooms.inject(0) { |sum, rooms| sum + rooms.sqft }
+  end
+
+  def price_per_sqft
+    (monthly_rent / total_sqft).round(1)
```

Comment:
in Ruby, division truncate the answer : eg 5/2 is 2...   you should change it to a float - monthly_rent / total_sqft.to_f